gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -20,7 +20,7 @@
 // arrival times.
 //
 // This protocol is published at:
-// https://developers.google.com/transit/gtfs-realtime/
+// https://github.com/google/transit/blob/master/gtfs-realtime/proto/gtfs-realtime.proto
```

Comment:
Just link to the gtfs-realtime directory of the project, rather than the proto file directly?

We should probably add a README.md directly at the gtfs-realtime directory level to quickly point to the most-relevant files/entry-points too (which will be especially helpful once there's more than an 'en' directory under /spec/).